code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
This isn't the solution we're looking for. Giving the detective security access will make them feel obligated to conduct arrests and do security's job. Furthermore, it turns the detective into nothing but a security officer with extra access to lucrative equipment. Can't let this stay.